test/role.test.js

```diff
@@ -87,6 +87,20 @@ describe('role model', function() {
     });
   });
 
+  it('should not allow duplicate role name', function(done) {
+    Role.create({ name: 'userRole' }, function(err, role) {
+      if (err) return done(err);
+      Role.create({ name: 'userRole' }, function(err, role) {
+        expect(err).to.exist;
+        expect(err).to.have.property('name', 'ValidationError');
+        expect(err).to.have.property('message');
+        expect(err.message).to.have.string('`name` already exists');
+        expect(err).to.have.property('statusCode', 422);
```

Comment:
Nah, its fine. It's just a nitpick. We actually need to fix all the other ones. There should never be two empty lines, just single empty lines that group logical items (the lines that are related) together. Like returning the error is separate from calling Role.create, etc.